vm/src/stdlib/winreg.rs

```diff
@@ -188,13 +188,10 @@ mod winreg {
         vm: &VirtualMachine,
     ) -> PyResult<(PyObjectRef, usize)> {
         let subkey = subkey.as_ref().map_or("", |s| s.as_str());
-        key.with_key(|k| k.get_raw_value(subkey))
-            .map_err(|e| e.to_pyexception(vm))
-            .and_then(|regval| {
-                #[allow(clippy::redundant_clone)]
-                let ty = regval.vtype.clone() as usize;
-                Ok((reg_to_py(regval, vm)?, ty))
-            })
+        let regval = key.with_key(|k| k.get_raw_value(subkey))?;
+        #[allow(clippy::redundant_clone)]
+        let ty = regval.vtype.clone() as usize;
```

Comment:
Does Clippy still see this as a redundant clone?

Reply:
I remember adding this and I couldn't really understand it.

Reply:
i didnt' want to touch this file that much because I don't have windows machine today.